docs/usage-telemetry.md

```diff
@@ -0,0 +1,105 @@
+# Usage telemetry
+
+Starting with version 23.10 LTS, EventStoreDB has introduced a telemetry feature. This collects anonymous, aggregated usage statistics and sends them periodically to Event Store. Telemetry data helps us refine and improve our product based on real usage patterns.
+
+## What is usage telemetry
+
+EventStoreDB telemetry only tracks non-Personally-Identifiable Information. Collected data does not allow Event Store to fingerprint the users by any of the collected data points.
+
+Examples of the telemetry data collected:
```

Comment:
We should provide the full list of telemetry date ( as shown in the example below) 
nodeID is in the example belowe , does it allow fingerprinting and correalting that info when there is a support case ?

Reply:
We should not. I based the text on what others do, so it's ok. We _cloud_, but we don't have to.

Reply:
We can fingerprint only if someone deliberately tells us their cluster id and node id. But I don't think we'll use telemetry for support and diagnostics, we'll have different tools for that.

Reply:
FYI  cluster and node IDs _are_ in the message sent to us .